packages/module/src/catalog/QuickStartTileHeader.tsx

```diff
@@ -38,9 +40,12 @@ const QuickStartTileHeader: React.FC<QuickStartTileHeaderProps> = ({
 
   return (
     <div className="pfext-quick-start-tile-header">
-      <Title headingLevel="h3" data-test="title" id={quickStartId}>
-        <QuickStartMarkdownView content={name} />
-      </Title>
+      <div className="pfext-quick-start-title-header__display-name">
+        <Title headingLevel="h3" data-test="title" id={quickStartId}>
+          <QuickStartMarkdownView content={name} />
+        </Title>
+        {action}
```

Comment:
What's your thoughts on creating the bookmark internally?  I was able to create some strange results passing in the button component not set to a link.  The alignment also looked good in some cases but if the title got to large it looked to low.  I would think we would want it to stick to the top.  Let me know your thoughts.

Reply:
I was thinking about it from the start as well, but I was not that big of a fan of it. Good point on style misalignment, how about we change this to an object with `actionIcon`, `onActionClick` and `actionAriaLabel`. Right now we want to show plain button. If someone would want to display a dropdown in here, with these 3 items they should be able to do that (by using dropdown menu and attaching it to the right place) should there be such need in the future.

Reply:
That works for me.  Maybe make one of the cards in the demo doc with how to implement it.  So people would know.